VK/VK/Operation/LoadDataGroupService.swift

```diff
@@ -0,0 +1,55 @@
+// LoadDataGroupService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Формирование запроса группы
+class LoadDataGroupService {
+    private enum Constants {
```

Comment:
```suggestion
    private enum Constants {
```
Я заметил ты где-то помечаешь константы маркой, а где-то -- нет. Я бы привел к общему виду)